hack/update-gofmt.sh

```diff
@@ -19,4 +19,4 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-find . -name "*.go" | grep -v "\/vendor\/" | xargs gofmt -s -w
+find . -name "*.go" | grep -v "\\/vendor\\/" | xargs gofmt -s -w
```

Comment:
they're technically not necessary since `/` is not a valid escape sequence char bash will take `\/` as literal, but shellcheck will complain that you should really make literal `\` with `\\` (which is a good idea in case you change `/`), they evaluate to the same thing